layouts/_default/single.html

```diff
@@ -1,43 +1,40 @@
 <!DOCTYPE html>
 <html>
-{{ partial "header.html" . }}
-<body>
-	<div class="content">
```

Comment:
This div makes the pages to be centered/aligned , to compensate for this div it's be great if you could change the css of content class to body.

Thanks